src/common/components/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
```

Comment:
from my point of view, it doesn't make lot's of sense to have `components` and `common/components` directories. it would make sense if you have some kind of separation based on the screens, (e.g. `movie info`, `movies list`, etc.). Also, you might check atomic design http://atomicdesign.bradfrost.com/chapter-2/#the-atomic-design-methodology